source/gwork/include/Gwork/Controls/Property/ColorSelector.h

```diff
@@ -106,7 +106,7 @@ namespace Gwk
                 {
                     ParentClass::DoChanged();
                     unsigned int r,g,b;
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(__MINGW32__)
```

Comment:
Since `sscanf_s` is a Visual Studio feature only, it probably would be better to just check whether we're using MSVC instead of "not MinGW".